client/src/stylings/aboutUs.css

```diff
@@ -0,0 +1,97 @@
+.pageDiv {
```

Comment:
For consistency, changing the directory name to singular "styling" would be better